pkgs/applications/misc/pot/default.nix

```diff
@@ -0,0 +1,120 @@
+{ lib
+, stdenv
+, stdenvNoCC
+, rustPlatform
+, fetchFromGitHub
+, wrapGAppsHook
+, cargo
+, rustc
+, cargo-tauri
+, pkg-config
+, nodePackages
+, jq
+, moreutils
+, libayatana-appindicator
+, gtk3
+, webkitgtk
+, libsoup
+, openssl
+, xdotool
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pot";
+  version = "0.4.4";
+
+  src = fetchFromGitHub {
+    owner = "pot-app";
+    repo = "pot-desktop";
+    rev = version;
+    hash = "sha256-RsxEd/UHtjvWYsPVukJUA3Wvp6auUd+X52+g5uN96jc=";
+  };
+
+  sourceRoot = "source/src-tauri";
+
+  postPatch = ''
+    substituteInPlace $cargoDepsCopy/libappindicator-sys-*/src/lib.rs \
+      --replace "libayatana-appindicator3.so.1" "${libayatana-appindicator}/lib/libayatana-appindicator3.so.1"
+
+    chmod -R +w ..
+    # Disable auto update check by default
+    sed -i -e '/auto_check/s/true/false/' src/main.rs ../src/windows/Config/index.jsx
+  '';
+
+  pnpm-deps = stdenvNoCC.mkDerivation {
+    pname = "${pname}-pnpm-deps";
+    inherit src version;
+
+    nativeBuildInputs = [
+      jq
+      moreutils
+      nodePackages.pnpm
+    ];
+
+    installPhase = ''
+      export HOME=$(mktemp -d)
+      pnpm config set store-dir $out
+      pnpm install
+
+      # Remove timestamp and sort the json files
+      rm -rf $out/v3/tmp
+      for f in $(find $out -name "*.json"); do
+        sed -i -E -e 's/"checkedAt":[0-9]+,//g' $f
+        jq --sort-keys . $f | sponge $f
+      done
+    '';
+
+    dontFixup = true;
+    outputHashMode = "recursive";
+    outputHash = "sha256-GHRgqsTwINEIqhnmcC/TaDrUDSuHXhl66kRy3T/XlT0=";
```

Comment:
That hash seems to depend on the platform. For aarch64-linux we get ``sha256-O3ynJScipCw1ORgmC1QgM07kkqA3svv9dYvlaqpoBNk=``.

Reply:
Looks like this is fixed.